coordinator/src/test/scala/filodb.coordinator/queryplanner/SingleClusterPlannerSpec.scala

```diff
@@ -386,6 +387,7 @@ class SingleClusterPlannerSpec extends AnyFunSpec with Matchers with ScalaFuture
   // Target-Schema start
 
   it("should stitch results when target-schema changes during query range") {
+    engine.invalidateCaches()
```

Comment:
You can add `implements BeforeAndAfterEach` to the test class then `override def beforeEach(): Unit` so this runs before every test.

Reply:
Good point,  let me do that.

Reply:
Pushed changes